front-end/angular.json

```diff
@@ -207,8 +207,8 @@
           "builder": "@cypress/schematic:cypress",
           "options": {
             "devServerTarget": "fecfile-web:serve",
-            "watch": true,
-            "headless": false
+            "watch": false,
```

Comment:
did you mean to update this?

Reply:
Good catch. Not at the time, but I do want it updated now along with headless: true